Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -305,6 +306,25 @@ theorem comp_congr (e : M' ≃ₗ[R] M'') (B : BilinForm R M) (l r : M' →ₗ[R
       B.comp (l.comp (e.symm : M'' →ₗ[R] M')) (r.comp (e.symm : M'' →ₗ[R] M')) :=
   rfl
 
+variable {N₁ N₂ : Type*}
+variable [AddCommMonoid N₁] [AddCommMonoid N₂] [Module R N₁] [Module R N₂]
+
+/-- When `N₁` and `N₂` are equivalent, bilinear maps on `M` into `N₁` are equivalent to bilinear
+maps into `N₂`. -/
+@[simps]
+def _root_.LinearMap.BilinMap.congr₂ (e : N₁ ≃ₗ[R] N₂) : BilinMap R M N₁ ≃ₗ[R] BilinMap R M N₂ where
```

Comment:
This looks quite similar to [LinearEquiv.congrRight](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Module/Equiv/Basic.html#LinearEquiv.congrRight); I think we can use that to implement this?

Reply:
Thanks. Done. In a similar vein, we can also implement `LinearMap.BilinForm.congr` as:
```
def congr (e : M ≃ₗ[R] M') : BilinForm R M ≃ₗ[R] BilinForm R M' :=
  LinearEquiv.congrRight (LinearEquiv.congrLeft _ _ e) ≪≫ₗ LinearEquiv.congrLeft _ _ e
```
I've pushed a separate commit with this, but perhaps there are reasons why it was previously done directly? (Performance?)